src/customer/strategies/googlepay/googlepay-customer-strategy.ts

```diff
@@ -123,7 +123,13 @@ export default class GooglePayCustomerStrategy extends CustomerStrategy {
 
         return this._googlePayPaymentProcessor.displayWallet()
             .then(paymentData => this._googlePayPaymentProcessor.handleSuccess(paymentData)
-                .then(() => this._googlePayPaymentProcessor.updateShippingAddress(paymentData.shippingAddress)))
+            .then(() => {
+                if (paymentData.shippingAddress) {
+                    this._googlePayPaymentProcessor.updateShippingAddress(paymentData.shippingAddress);
+                }
+
+                return Promise.resolve();
```

Comment:
You don't need this line. Can you refer to my previous comment if you want to know the reason?